README.md

```diff
@@ -1,42 +1,68 @@
-# Turborepo starter with pnpm
+<p align="center">
+ <img src="https://user-images.githubusercontent.com/9203826/154288895-670f5c23-81a1-4307-a080-1af83f7f8356.svg" align="center" alt="Ledger" />
+ <h2 align="center">Platform API</h2>
```

Comment:
Quick question, do we rename everything Wallet API ?

Reply:
I think we can rename everything Wallet API. It seems to be the de facto name of the project